docbrown/src/db/path.rs

```diff
@@ -10,7 +11,7 @@ use std::ops::Range;
 use std::sync::Arc;
 
 #[derive(Copy, Clone)]
-pub(crate) enum Operations {
+pub enum Operations {
```

Comment:
why is this needed?

Reply:
After pairing for a while, the conclusion of this is that ideally `DynamicGraph::window` should return `DynamicGraph` instead of `WindowedGraph`. That way we could avoid doing translations that force us to make some things public, such as `Operations`. However, we tried it and it seems impossible or at least very hard with the approach we followed. It might be feasible some other way though, so we can try to improve this later.